plugin/test/position.js

```diff
@@ -0,0 +1,29 @@
+import { getTestSuite } from './_helpers.js'
+import * as assert from 'uvu/assert'
+
+const test = getTestSuite('position')
+
+test('order allows values 1 to 12', async (t) => {
+  const range = [...Array(12).keys()].map(x => ++x)
```

Comment:
Smallest possible nit - this 👇  feels a bit more clear than spreading `keys` into an array. Totally feel free to ignore, but I had to stop and think for a sec about what was going on there.

```
Array.from({ length: 12 }).map((_, i) => i + 1)
```

Note: `++i` would be fine here instead of `i + 1`, just used to people preferring the `i + 1` style 🤷

Reply:
Im not one of those thats against nitpicking, Im one of those that think that all of those small little nit picks actually adds to the greater good :) Will most dev look into your suggestion 🙏